accepted/0018-notification-widget.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: 2020-04-17
+- RFC PR: #18
+- Mantis Issue: N/A
+
+- Related PRs:
+	1. https://github.com/obsproject/obs-studio/pull/1832
+
+# Summary
+
+Add the ability to display notifications to the user.
+
+# Motivation
+
+Currenty, there is no way of showing custom notifications to users.
+
+# Detailed design
+
+## User UX
+
+- Display notifications at top of main window
+- Have three notification types (Info, Error, and Warning)
```

Comment:
Could you add some details to this RFC on how your PR https://github.com/obsproject/obs-studio/pull/1832 used these, and what you'd think these could be used for? E.g. when would you use an "Info" notification (concrete example please, not just "something that happens but doesn't require immediate user action")